fellowship/cli.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2021 Nokia
+# Licensed under the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import argparse
+import json
+import sys
+from .contract_generator import ContractGenerator
+from .rest_tester import RestTester
+
+
+def validation(args):
+    rest_tester = RestTester(args.path)
+    rest_tester.make_requests_and_validates()
+
+
+def generation(args):
+    contract_generator = ContractGenerator(args.path)
+    contract_generator.generate_and_save_contract(args)
+
+
+def parse_args(args):

```

Comment:
The order here could be different. First parse_args and then sub-parser functions. Maybe these sub-parser function could be even protected?